modules/virt-add-disk-to-vm.adoc

```diff
@@ -38,6 +38,10 @@ Use this procedure to add a virtual disk to a {object}.
 
 . In the *Add Disk* window, specify the *Source*, *Name*, *Size*, *Type*, *Interface*, and *Storage Class*.
 
+.. Optional: Select the *Enable preallocation* checkbox to improve write performance when creating data volumes. However, this option should only be used if you really need maximum write performance.
+
+... NOTE: Enabling preallocation is available only for a blank disk source.
```

Comment:
A couple of things:

Even though the `NOTE:` format is valid in asciidoc, the [OCP contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#admonitions) specify the following format:
```
[NOTE]
====
<note_content>
====
```

(When the guidelines specify one syntax convention over another, it's usually to avoid issues building on the portal.)

The `...` bullet is not necessary here. You can put a `+` on the line above the note so that it is in line with the *Enable preallocation* step.

Reply:
On second thought, I might nix the note entirely and put "If you use a blank disk source, you can..." at the beginning of the `Optional:` step.